<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Frequency</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Audio Input</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="audioInputComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Display Mode</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="displayModeLayout">
       <item>
        <widget class="QComboBox" name="displayModeComboBox">
         <item>
          <property name="text">
           <string>Normalized</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Standard</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minDecibelsSpinBox">
         <property name="suffix">
          <string> dB</string>
         </property>
         <property name="prefix">
          <string>Min: </string>
         </property>
         <property name="minimum">
          <number>-100</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>-80</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="maxDecibelsSpinBox">
         <property name="suffix">
          <string> dB</string>
         </property>
         <property name="prefix">
          <string>Max: </string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>80</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Window Type</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="windowFunctionComboBox">
       <item>
        <property name="text">
         <string>Hanning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blackman</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blackman-Harris</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangular (None)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="minFrequencySpinBox">
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="prefix">
          <string>Min: </string>
         </property>
         <property name="maximum">
          <number>44100</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="maxFrequencySpinBox">
         <property name="suffix">
          <string> Hz</string>
         </property>
         <property name="prefix">
          <string>Max: </string>
         </property>
         <property name="maximum">
          <number>44100</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
